shared/constants/chat2/message.js

```diff
@@ -922,6 +926,12 @@ export const messageExplodeDescriptions: Types.MessageExplodeDescription[] = [
   {text: 'Never explode (turn off)', seconds: 0},
 ].reverse()
 
+export const getFilename = (path: string) => {
```

Comment:
maybe you should use helpers from constants/fs for this

Reply:
Eh, this works well enough for what it needs to do. It was just hanging out in some container before.

Reply:
but you can just call getLocalPathName  right?

Reply:
ok